### PR TITLE
add duply image & tests

### LINKED ISSUE #1: add duply image
This should be similar to [docker-duply](https://github.com/scalableminds/docker-duply).
Things that should be changed:
* don't include mongo
* consider not sending mails
* rename `HOSTPATH` to something more useful
* use alpine (https://pkgs.alpinelinux.org/package/edge/main/x86_64/duply)

### LINKED ISSUE #2: add docker projects
Move the following projects here and [ugrade to circleCI 2](https://circleci.com/sunset1-0/):
* [x] docker-sbt-node-mongo (scalableminds/docker-sbt-node-mongo#16)
* [x] openjdk base image with Berlin timezon
* [x] docker-helm (#5)
* [x] docker-kubectl
* [ ] docker-duply (#1)

### PR DESCRIPTION
**Pull request by jstriebel**
fixes #1
fixes #2

### CODE REVIEW

File: duply/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM debian:buster
```
Comment (jstriebel) [inline duply/Dockerfile]: The version that comes with debian stretch `1.11.3` has a bug with `file://` backups, which we use in the test. This is fixed in `2.0.2` and included in debian buster.
See https://duply.net/wiki/index.php/Duply-Changelog

File: duply/conf.template [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# the following is an incomplete list (<backend>: comma separated env vars list)
+# Azure: AZURE_ACCOUNT_NAME, AZURE_ACCOUNT_KEY
+# Cloudfiles: CLOUDFILES_USERNAME, CLOUDFILES_APIKEY, CLOUDFILES_AUTHURL
+# Google Cloud Storage: GS_ACCESS_KEY_ID, GS_SECRET_ACCESS_KEY
+# Pydrive: GOOGLE_DRIVE_ACCOUNT_KEY, GOOGLE_DRIVE_SETTINGS
+# S3: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+# Swift: SWIFT_USERNAME, SWIFT_PASSWORD, SWIFT_AUTHURL,
+#        SWIFT_TENANTNAME OR SWIFT_PREAUTHURL, SWIFT_PREAUTHTOKEN
+export AWS_ACCESS_KEY_ID="$USER"
+export AWS_SECRET_ACCESS_KEY="$PASSWORD"
```
Comment (jstriebel) [inline duply/conf.template]: fixes https://github.com/scalableminds/docker-duply/issues/2

**Comment by jstriebel** [comment]
luckily we already have higher build numbers than before: https://hub.docker.com/r/scalableminds/duply/tags/

File: duply/Dockerfile (hunk 1 above)
Comment (normanrz) [reply duply/Dockerfile]: :+1:

**Pull request merged**